web/app/scripts/custom-reports/custom-reports-modal-controller.js

```diff
@@ -31,7 +31,7 @@
             ctl.geoAggSelected = null;
 
             // Add the active filters for display
-            ctl.nonDateFilters = _.omit(FilterState.filters, '__dateRange');
+            ctl.nonDateFilters = _.omit(FilterState.filters, ['__dateRange', '__createdRange']);
```

Comment:
Not a huge deal, but is there a constant we could reference instead of the string literals?

Reply:
Do you know if we have an example of sharing a constant like that across multiple files I can run off of? I'm totally down for less magic string literals, but I don't think I've seen a good example of how it'd look in JS.

Reply:
Yeah, I'm a bit worried that given my rustiness with AngularJS this may be something that's fairly easy to get wrong, especially with grunt and all that jazz. Worth keeping in mind in the future, though.

Reply:
The most AngularJs-y way that I can think of to do this would be to add them to the `FilterState` service or a similar service and pull them in as constants that way. Definitely not a big deal though.